charts/provisioner/templates/extra-list.yaml

```diff
@@ -0,0 +1,4 @@
+{{- range .Values.extraDeploy }}
+---
+{{ include "common.tplvalues.render" (dict "value" . "context" $) }}
+{{- end }}
```

Comment:
Not sure why this is needed?

Reply:
to allow deploy of other resources alongside this chart like ConfigMap/Secrets 
used in tests and examples